TransformationSystem/Client/TaskManager.py

```diff
@@ -337,7 +338,7 @@ def getSubmittedTaskStatus( self, taskDicts ):
 
       newStatus = self.requestClient.getRequestStatus( taskDict['ExternalID'] )
       if not newStatus['OK']:
-        log = self._logVerbose if 'not exist' in newStatus['Message'] else self.__logWarn
+        log = self._logVerbose if 'not exist' in newStatus['Message'] else self._logWarn
```

Comment:
Thanks... Grrrr.... I don't understand why the code checker didn't pick up that typo?